src/util/Collection.js

```diff
@@ -362,6 +362,61 @@ class Collection extends Map {
       return testVal !== value || (testVal === undefined && !collection.has(key));
     });
   }
+  
+  /**
+   * Sort a collection based on a function given
+   * @param {Function} fn Function used for comparing two objects
+   * @returns {Collection}
+   */
+   sort = (fn) {
```

Comment:
You're right. I probably missed that when copying from prototype.sort = function(fn). Sorry about that.